### PR TITLE
fix(UI): Poller and Remote server wizard render looks broken

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This PR fixes Poller / Remote server wizard background color.

**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

- Go to Configuration > Pollers then click on the button to add poller or remote server through a wizard.
- Each steps about remoter server and poller should be surrounded by a white background color.

## Checklist

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/front_src/src/components/forms/baseWizard.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+  page: {
+    backgroundColor: '#FFF',
```
Comment (bdauria) [inline www/front_src/src/components/forms/baseWizard.tsx]:
```suggestion
const useStyles = makeStyles((theme) => ({
  page: {
    backgroundColor: theme.palette.common.white,
```

**Comment by bdauria** [comment]
@Thebarda it seems that your changes weren't pushed

File: www/front_src/src/components/forms/baseWizard.tsx (hunk 1 above)
Comment (jdelpierre) [reply www/front_src/src/components/forms/baseWizard.tsx]: It's not pushed ?

Comment (kduret) [reply www/front_src/src/components/forms/baseWizard.tsx]: Indeed, seems not pushed

**Pull request merged**